outputs/nexusiq.go

```diff
@@ -0,0 +1,183 @@
+package outputs
```

Comment:
It looks like this is a REST API integration - could we add some unit tests that can guarantee the behaviour of this code?

Reply:
done